lib/src/fitness.g.dart

```diff
@@ -12,11 +12,30 @@ part of pana.health;
 
 Fitness _$FitnessFromJson(Map<String, dynamic> json) => new Fitness(
     (json['magnitude'] as num)?.toDouble(),
-    (json['shortcoming'] as num)?.toDouble());
+    (json['shortcoming'] as num)?.toDouble(),
+    suggestions: (json['suggestions'] as List)
+        ?.map((e) => e == null
```

Comment:
Do we want to keep `null` suggestions? Otherwise we could remove them first with a `where()`.

Reply:
Ah, this is generated code... Never mind!